### PR TITLE
ignores error 409

### PR DESCRIPTION
**Pull request by FredrikWartenberg**
- Allows to specify a list of http errors to be ignored
- Ignores 409 errors
- steps API Version to be tested against
- API version to be tested against will now be fetched from MVG class

### CODE REVIEW

File: mvg/mvg.py [hunk 1]
```diff
@@ -46,8 +46,11 @@ def __init__(self, endpoint: str, token: str):
         self.endpoint = endpoint
         self.token = token
 
-        self.mvg_version = self.parse_version("v0.4.1")
-        self.tested_api_version = self.parse_version("v0.1.0")
+        self.mvg_version = self.parse_version("v0.5.0")
```
Comment (SergioMDCB) [inline mvg/mvg.py:49]: This is more of a personal question, but had we defined somewhere when a change deserves to upgrade version in the last decimal, the middle one or the first one?

Comment (tuix) [reply mvg/mvg.py:49]: This is following semantic versioning convention (semver). It is explained [here](https://semver.org/)

**Review by SergioMDCB — APPROVED** [review]
This PR updates the MVG class by enabling to ignore error 409 via a subclass. In addition, updates the version test function.

File: mvg/mvg.py (hunk 1 above)
Comment (SergioMDCB) [reply mvg/mvg.py:49]: Thanks a lot for the answer

**Review by SergioMDCB — APPROVED** [review]
Documentation has been updated.

**Pull request merged**